### PR TITLE
Add explanatory sentence in endpoints section

### LINKED ISSUE #418: Add explanatory sentence in endpoints section
## Motivation

In all module LIPs, there is a section with title "Endpoints for Off-Chain Services". This section does not include all possible endpoints used in the implementation, especially not the trivial ones or the ones needed by UI tools.

## Current Specifications

Currently it is not explained that not all implemented endpoints  are included in the LIP specifications.

## Proposed Specifications

In the beginning of the section, add a sentence: " This section specifies the non-trivial or recommended endpoints of the module and does not include all endpoints. "
## Affected LIPs

All LIPs defining a module.

### PR DESCRIPTION
**Pull request by gkoumout**
Fixes #418

### CODE REVIEW

**Review by sergeyshemyakov — COMMENTED** [review]
In LIP 0048 I would write "This section specifies the non-trivial or recommended endpoints of the Fee module and does not include all endpoints. To be completed by the dev team."

In LIPs 0051/0052 I would write: "This section specifies the non-trivial or recommended endpoints of the Token/NFT module and does not include all endpoints. TBA."

Add the sentence to LIP 0057 also. Even though the LIP defined many endpoints, I would still have this sentence so developers are free to add whatever endpoints they might need in the future.

**Review by mjerkov — CHANGES_REQUESTED** [review]
What about LIP 0057, I'm not sure if it's not here by purpose or it's just a miss?

Also, maybe the last part of the sentence "and does not include all endpoints" can be restated as "and does not include all **possible** endpoints.", for the sake of clarity, although I don't insist in any sense.

File: proposals/lip-0051.md [hunk 1]
```diff
@@ -2625,7 +2625,7 @@ def beforeCrossChainMessageForwarding(trs: Transaction, ccm: CCM, ccmFailed: boo
 
 ### Endpoints for Off-Chain Services
 
-TBA
+This section specifies the non-trivial or recommended endpoints of the Token/NFT module and does not include all endpoints. TBA
```
Comment (sergeyshemyakov) [inline proposals/lip-0051.md:2628]: Here it should be just Token module.

File: proposals/lip-0052.md [hunk 2]
```diff
@@ -2294,7 +2294,7 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
 
 ### Endpoints for Off-Chain Services
 
-TBA
+This section specifies the non-trivial or recommended endpoints of the Token/NFT module and does not include all endpoints. TBA
```
Comment (sergeyshemyakov) [inline proposals/lip-0052.md:2297]: Here it should be just NFT module.

**Review by sergeyshemyakov — APPROVED** [review]
Accepted!

**Review by mjerkov — APPROVED** [review]
All good!

**Review by janhack — APPROVED** [review]
Editorial approval

**Pull request merged**